README.md

```diff
@@ -138,6 +139,32 @@ Function arguments are:
 - **`tlsVerify`** (defaults to `true`)
 
 
+### `nix2container.pullImageByManifest`
+
+Function arguments are:
```

Comment:
Because this function is not known in dockerTools, i think it would be nice to briefly explain how it works and its advantages.

Reply:
Makes sense; done as of 16a79fcbcb4571228b3efb5c811a505224c9ce6e.